jniFaiss/CMakeLists.txt

```diff
@@ -0,0 +1,108 @@
+cmake_minimum_required(VERSION 2.8)
+
+project(KNNIndex_FAISS)
```

Comment:
I think this might be better for CMakeLists.txt: https://gist.github.com/jmazanec15/da1e68aed9073c7d4706597ba8ef2087.

Reply:
nice code! i'll use this one